src/Modal/FullscreenModal.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ModalDialog from './ModalDialog';
+
+const FullscreenModal = ({ children, footerNode, ...props }) => (
+  <ModalDialog {...props}>
+    <ModalDialog.Header>
+      <ModalDialog.Title>{props.title}</ModalDialog.Title>
+    </ModalDialog.Header>
+    <ModalDialog.Body>{children}</ModalDialog.Body>
+    {footerNode && <ModalDialog.Footer>{footerNode}</ModalDialog.Footer>}
+  </ModalDialog>
+);
+
+FullscreenModal.propTypes = {
+  children: PropTypes.node.isRequired,
+  title: PropTypes.string.isRequired,
+  onClose: PropTypes.func.isRequired,
+  isOpen: PropTypes.bool,
+  hasCloseButton: PropTypes.bool,
+  size: PropTypes.oneOf(['sm', 'md', 'lg', 'xl', 'fullscreen']),
+  variant: PropTypes.oneOf(['default', 'warning', 'danger', 'success', 'dark']),
```

Comment:
OK, following up on my comment above, how about yes, let's throw these and the sized in a constant somewhere, we're gonna reuse them apparently. 😉